vta/python/vta/top/graphpack.py

```diff
@@ -246,32 +246,42 @@ def visit_call(self, call):
 
 class BT(Exception):
     pass
-def get_subgraph(expr, start_name, stop_name):
+def get_subgraph(expr, start_name, stop_name, start_name_indx=0, stop_name_indx=0):
```

Comment:
is this a typo? the abbreviation for 'index' should be either 'ind', 'idx', or 'index'.

Reply:
Agree, please use `idx`

Reply:
fixed